HunterPie.UI/Overlay/Widgets/Wirebug/WirebugContextHandler.cs

```diff
@@ -37,14 +37,14 @@ private void OnTimerUpdate(object sender, MHRWirebug e)
     {
         MaxTimer = e.MaxTimer;
         Timer = e.Timer;
-        IsTemporary = Context.Timer > 0;
+        IsTemporary = Timer > 0;
     }
 
     private void OnAvailable(object sender, MHRWirebug e) => IsAvailable = e.IsAvailable;
 
     private void OnCooldownUpdate(object sender, MHRWirebug e)
     {
-        MaxCooldown = e.MaxCooldown;
+        MaxCooldown = e.MaxCooldown == 0 ? 400 : e.MaxCooldown;
```

Comment:
I believe this should be fixed in the core rather than in the UI, otherwise future plugins would need to implement this fix too if it's not fixed in the core.

Reply:
Fixed MHRWirebugData instead of ViewModel.